spec/TableUtils_spec.lua

```diff
@@ -1,5 +1,17 @@
 local TableUtils = require "TableUtils"
 
+local function lazySequence(firstNumber, lastNumber)
+	local i = -1
+	return function()
+		i = i + 1
```

Comment:
Is there a reason for incrementing before you've even used the variable? Why not just initialise `i` to 0?

Reply:
saves having an i = i + 1 before each return, seems very minor to warrant any changes

Reply:
You only need that on one branch. Could do something like local nextNumber = firstNumber + i

if nextNumber > lastNumber then
	return
else
	local nextIndex = i
	i = i + 1
	return nextIndex, nextNumber
end`